app/exporters/hathi/submission_information_package.rb

```diff
@@ -12,52 +14,60 @@ def initialize(package:, base_path:)
       @checksums = {}
       @digester = Digest::MD5.new
     end
-
+    
     def export
-      Zip::File.open(File.join(base_path, package.id.to_s + ".zip"),
-                     Zip::File::CREATE) do |zipfile|
-        deposit_files(zipfile)
-        deposit_metadata(zipfile)
-        deposit_checksums(zipfile)
-      end
+      @export_dir = FileUtils.mkdir(
+      File.join(base_path, package.id.to_s + "_sip"),
+      :mode => 0700)
+      deposit_files()
+      deposit_metadata()
+      deposit_checksums()
     end
-
+    
     private
-
-      def deposit_files(zipfile)
-        package.pages.each do |page|
+    
+    def deposit_files
+      package.pages.each do |page|
+        digester.reset
+        FileUtils.cp page.tiff_path, 
+          File.join(@export_dir, page.name + ".tif")
+        digester.file page.tiff_path
+        checksums[page.name + ".tif"] = digester.hexdigest
+        
+        if page.ocr?
           digester.reset
-          zipfile.add(page.name + ".tif", page.tiff_path)
-          digester.file page.tiff_path
-          checksums[page.name + ".tif"] = digester.hexdigest
-
-          if page.ocr?
-            digester.reset
-            zipfile.get_output_stream(page.name + ".txt") { |f| f.write page.to_txt }
-            digester << page.to_txt
-            checksums[page.name + ".txt"] = digester.hexdigest
+          File.open(File.join(@export_dir, page.name + ".txt"), "w")  do |f|
+            f.write page.to_txt
           end
-
+          digester << page.to_txt
+          checksums[page.name + ".txt"] = digester.hexdigest
+          
           next unless page.hocr?
+
           digester.reset
-          zipfile.get_output_stream(page.name + ".html") { |f| f.write page.to_html }
+          File.open(File.join(@export_dir, page.name + ".html"), "w") do |f|
+            f.write page.to_html
+          end
           digester << page.to_html
           checksums[page.name + ".html"] = digester.hexdigest
         end
       end
-
-      def deposit_checksums(zipfile)
-        zipfile.get_output_stream("checksum.md5") do |f|
-          # iterate over k,v in checksums
-          checksums.each { |k, v| f.write format("%s %s\n", v, k) }
-        end
+    end
+    
+    def deposit_checksums()
+      File.open(File.join(@export_dir, "checksum.md5"), "w") do |f|
+        # iterate over k,v in checksums
+        checksums.each { |k, v| f.write format("%s %s\n", v, k) }
```

Comment:
Out of curiosity, does Hathi allow more than one space in the checksum file? If they allow it, I'd recommend using two spaces, since that allows the standard Unix `md5sum` command to verify the checksums.